prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -954,5 +954,75 @@ public async Task<IEnumerable<string>> GetEnrolleeEmails(BulkEmailType bulkEmail
                 .DecompileAsync()
                 .ToListAsync();
         }
+
+        public async Task<EnrolleeAbsence> CreateEnrolleeAbsenceAsync(int enrolleeId, DateTime startTimestamp, DateTime endTimestamp)
+        {
+            if (startTimestamp >= endTimestamp)
+            {
+                throw new ArgumentException($"endTimestamp is not after the startTimestamp of absence.");
+            }
+
+            var enrollee = await _context.Enrollees
```

Comment:
You don't need to pull up the `Enrollee`, you can just add the new absences to `_context.EnrolleeAbsences`

Reply:
whoops, artifact of the first implementation with not using the context.